docs/source/notes/autograd.rst

```diff
@@ -245,49 +279,133 @@ where
             \frac{\partial u(x, y)}{\partial x} & \frac{\partial u(x, y)}{\partial y}\\
             \frac{\partial v(x, y)}{\partial x} & \frac{\partial v(x, y)}{\partial y} \end{bmatrix} \\
 
-This is similar to the definition of the JVP for a function defined from :math:`R^2 → R^2`, and the multiplication
-with :math:`[1, 1j]^T` is used to identify the result as a complex number.
+The key assumption made in the derivation of :eq:`[3]` is that function :math:`F` is a part of a
+real valued function. It implies that the Conjugate Wirtinger and Wirtinger derivative flowing back
+from the last `ℂ -> ℝ` function in :math:`G`, say :math:`H`, are complex conjugates of each other.
+This allows us to use :math:`grad\_output` as the incoming Conjugate Wirtinger derivative and recover the
+incoming Wirtinger derivative by complex conjugation of :math:`grad\_output` for function :math:`H`.
+For more details, please check out Section 3.4 `here <https://arxiv.org/pdf/1701.00392.pdf>`_.
```

Comment:
Is this easier to understand, when you define 
`dJ/ds* = dJ/du + dJ/dv j` and `dJ/ds = dJ/du - dJ/dv j`.
Then it is obvious, that `dJ/ds* = (dJ/ds)*`.